packages/contracts-bedrock/scripts/DeployAuthSystem.s.sol

```diff
@@ -54,3 +54,34 @@ contract DeployAuthSystemInput is CommonBase {
         return _owners;
     }
 }
+
+contract DeployAuthSystemOutput is CommonBase {
+    Safe internal _safe;
+
+    // This method lets each field be set individually. The selector of an output's getter method
+    // is used to determine which field to set.
```

Comment:
Same as https://github.com/ethereum-optimism/optimism/pull/11889#discussion_r1757676463 since not including them makes maintenance easier if we ever change the comments

Reply:
Agree

f15f3672b876a7a8f3696cc15eb31181d3f13a23